guides/common/modules/proc_configuring-host-based-authentication-control.adoc

```diff
@@ -1,62 +1,92 @@
-[id="Configuring_Host_Based_Authentication_Control_{context}"]
-= Configuring host-based authentication control
+[id="Configuring_Host_Based_Access_Control_{context}"]
+= Configuring host-based access control for {FreeIPA} users logging in to {Project}
 
-HBAC rules define which machine within the domain a {FreeIPA} user is allowed to access.
-You can configure HBAC on the {FreeIPA} server to prevent selected users from accessing {ProjectServer}.
-With this approach, you can prevent {Project} from creating database entries for users that are not allowed to log in.
+In {FreeIPA}, HBAC rules define which {FreeIPA} users or user groups can access specified {FreeIPA} hosts or host groups by using which {FreeIPA} services or services in a service group.
+For example, you can configure HBAC on the {FreeIPA} server to limit access to {ProjectServer} only to selected users or user groups.
+Using HBAC rules ensures {Project} does not create database entries for users who are not allowed to log in.
+
+.Procedure
+
+. On the {FreeIPA} server, configure HBAC control.
++
+For more information on HBAC in {FreeIPA}, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
++
+.. Authenticate as a user with privileges required to configure HBAC rules.
+.. Create a HBAC service for {ProjectServer}.
+.. Create a new HBAC HBAC rule to define the required access control.
+.. Add the HBAC service to the HBAC rule.
+.. To the HBAC service, add the user or user group whose access you want to specify and the hostname of your {ProjectServer}.
+.. Make sure the default {FreeIPA} `allow_all` rule is disabled.
 ifndef::orcharhino[]
-For more information on HBAC, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
 endif::[]
+. On your {ProjectServer}, re-run {foreman-installer} and define the HBAC service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-pam-service=_ipa-hbac-service_
```

Comment:
Why would you change the pam service name to something other than the default `foreman`? We explicitly overwrite the service with https://github.com/theforeman/puppet-foreman/blob/master/templates/pam_service.erb so if you use some existing pam service (other than `foreman`) I think you're likely to break something.

Whether that overwriting is correct, is a good question but it's the code we've been shipping since https://github.com/theforeman/puppet-foreman/commit/378d602424da89f4493767949d534b66942db8e2, which was merged 10 years and 2 days ago.

Reply:
Good timing! I just found this out myself. Crucially, the HBAC service on the IPA side needs to be named `satellite-prod` too, otherwise access is not granted. This is important because it means we need to be explicit about the fact that the service name is not a replaceable. (Maybe it's just me I don't think that was entirely clear before, hence my mistake.)